docs/syscall_descriptions.md

```diff
@@ -187,6 +190,22 @@ operand value interception and list of typical magic values.
 
 Note: some values for flags may be undocumented only as an oversight. These values should be added to descriptions.
 
+<div id="flags"/>
+
+### Flags/enums
+
+The `flags` type is used for all of:
```

Comment:
Maybe the name "flags" is the source of confusion? Is there a better name for just any set of values? "enum" matches C terminology, which is good, but doesn't it have the opposite meaning skew ("one of")?...